force-app/main/default/lwc/recordPickerDynamicTarget/recordPickerDynamicTarget.html

```diff
@@ -0,0 +1,41 @@
+<template>
+    <lightning-card
+        title="RecordPickerDynamicTarget"
+        class="slds-card__body_inner"
+        icon-name="standard:search"
+    >
+        <div class="slds-form-element">
+            <label class="slds-form-element__label">{label}</label>
```

Comment:
General comment that applies in multiple places: let's just hardcode the labels and placeholders in the template since they are static. We won't translate them and we don't need to pass them from a parent component. Unlike most components recipes are meant to be standalone and not reusable.
This will remove some lines from the JS and help learners to focus on the dynamic parts.